doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -328,49 +328,62 @@ applied from the mid till the end of the beam.
->>> R1, R2 = symbols('R1, R2')
->>> b = Beam(6, E, I)
->>> b.apply_load(R1, 0, -1)
->>> b.apply_load(1.5, 3, -2)
->>> b.apply_load(-3, 3, 0)
->>> b.apply_load(-1, 3, 1)
->>> b.apply_load(R2, 6, -1)
->>> b.bc_deflection.append((0, 0))
->>> b.bc_deflection.append((6, 0))
->>> b.solve_for_reaction_loads(R1, R2)
->>> b.reaction_loads
-    {R₁: 2.75, R₂: 10.75}
->>> b.load
-            -1              -2            0          1                -1
-    2.75⋅<x>   + 1.5⋅<x - 3>   - 3⋅<x - 3>  - <x - 3>  + 10.75⋅<x - 6>
->>> b.shear_force()
-                                                    2
-            0              -1            1   <x - 3>                 0
-    2.75⋅<x>  + 1.5⋅<x - 3>   - 3⋅<x - 3>  - ──────── + 10.75⋅<x - 6>
-                                                2
->>> b.bending_moment()
-                                        2          3
-            1              0   3⋅<x - 3>    <x - 3>                 1
-    2.75⋅<x>  + 1.5⋅<x - 3>  - ────────── - ──────── + 10.75⋅<x - 6>
-                                   2           6
->>> b.slope()
-                                                      4
-             2              1              3   <x - 3>                 2
-    1.375⋅<x>  + 1.5⋅<x - 3>  - 0.5⋅<x - 3>  - ──────── + 5.375⋅<x - 6>  - 15.6
-                                                  24
-    ───────────────────────────────────────────────────────────────────────────
-                                        E⋅I
+.. plot::
+   :context: close-figs
+   :format: doctest
+   :include-source: True
+   
+   >>> from sympy.physics.continuum_mechanics.beam import Beam
+   >>> from sympy import symbols, plot, S
+   >>> E, I = symbols('E, I')
+   >>> R1, R2 = symbols('R1, R2')
+   >>> b = Beam(6, E, I)
+   >>> b.apply_load(R1, 0, -1)
+   >>> b.apply_load(-S(3)/2, 3, -2)
+   >>> b.apply_load(3, 3, 0)
+   >>> b.apply_load(1, 3, 1)
+   >>> b.apply_load(R2, 6, -1)
+   >>> b.bc_deflection.append((0, 0))
+   >>> b.bc_deflection.append((6, 0))
+   >>> b.solve_for_reaction_loads(R1, R2)
+   >>> b.reaction_loads
+      {R₁: -11/4, R₂: -43/4}
 
->>> b.deflection()
-                                                                               5
-                                   3               2                4   <x - 3>                            3
-    -15.6⋅x + 0.458333333333333⋅<x>  + 0.75⋅<x - 3>  - 0.125⋅<x - 3>  - ──────── + 1.79166666666667⋅<x - 6>
-                                                                          120
-    ────────────────────────────────────────────────────────────────────────────────────────────────────────
-                                                      E⋅I
+   >>> b.load
+            -1            -2                                     -1
+      11⋅<x>     3⋅<x - 3>              0          1   43⋅<x - 6>  
+    - ──────── - ─────────── + 3⋅<x - 3>  + <x - 3>  - ────────────
+         4            2                                     4     
+   >>> plot(b.load)  # doctest: +SKIP
```

Comment:
You'll need to separate the directive here, so the plot shows after this line, right?

Reply:
yes

Reply:
> You'll need to separate the directive here, so the plot shows after this line, right?

I dont get it
Do i have to add this after line 356?
`.. plot::
   :context: close-figs
   :format: doctest
   :include-source: True`

Reply:
Probably after 358. I think we show intermediate plots in the docstring examples for reference.

Reply:
Example 1 does it.